package/gluon-client-bridge/files/lib/gluon/upgrade/320-gluon-client-bridge-wireless

```diff
@@ -21,14 +21,20 @@ local function configure_client(config, radio, index, suffix)
 
   uci:delete('wireless', name)
 
+  if index == 0 then
+    macaddr = util.generate_mac(0)
+  elseif index == 1 then
+    macaddr = util.generate_mac(3)
+  end
+
```

Comment:
Let's go with the following:

Drop the `% 8` from *generate_mac*, instead just return *nil* when *i* is too big. Change the functions which may use invalid indexes not to generate a config section at all when *generate_mac* returned *nil*.